AutomatedTesting/Gem/PythonTests/largeworlds/landscape_canvas/EditorScripts/GraphClosed_OnLevelChange.py

```diff
@@ -73,7 +72,7 @@ def GraphClosed_OnLevelChange():
     Report.result(Tests.graph_registered, graph_registered)
 
     # Open a different level, which should close any open Landscape Canvas graphs
-    general.open_level_no_prompt('WhiteBox/EmptyLevel')
+    general.open_level_no_prompt('Base')
```

Comment:
The comment says "open a different level", but isn't opening 'Base' opening the *same* level?  Does it matter?

Reply:
It's a different level. They have the same name but in different folders. hydra.open_base_level() opens Prefab/Base. This line just opens Base at the levels directory.